docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -114,43 +129,46 @@ export IMAGE_NAME="rhcos"
 export FLAVOR="m1.large"
 
 # Optional flags:
-# External network to use for the API and Ingress endpoints.
-export EXTERNAL_ID="5387f86a-a10e-47fe-91c6-41ac131f9f30"
+# External network to use for the Ingress endpoint.
+export EXTERNAL_NETWORK_ID="5387f86a-a10e-47fe-91c6-41ac131f9f30"
 
 # CA certificate path to use for the OpenStack API if using self-signed certificates.
+# In 4.18, this is not required as the CA cert found in clouds.yaml will be used.
 export CA_CERT_PATH="$HOME/ca.crt"
 
 # SSH Key for the nodepool VMs
 export SSH_KEY="$HOME/.ssh/id_rsa.pub"
 
 hcp create cluster openstack \
 --name $CLUSTER_NAME \
+--base-domain $BASE_DOMAIN \
 --node-pool-replicas $WORKER_COUNT \
 --pull-secret $PULL_SECRET \
 --ssh-key $SSH_KEY \
 --openstack-ca-cert-file $CA_CERT_PATH \
---openstack-external-network-id $EXTERNAL_ID \
+--openstack-external-network-id $EXTERNAL_NETWORK_ID \
 --openstack-node-image-name $IMAGE_NAME \
 --openstack-node-flavor $FLAVOR
 ```
 
 !!! note
 
-    A default NodePool will be created for the cluster with 2 vm worker replicas
+    A default NodePool will be created for the cluster with 2 VM worker replicas
     per the `--node-pool-replicas` flag.
 
 !!! note
 
-    To enable HA, the `--control-plane-availability-policy` flag can be set to `HighlyAvailable`.
-    This requires at least 3 worker nodes in the management cluster.
+    When using `hcp` CLI, High Availability will be enabled by default.
     Pods will be scheduled across different nodes to ensure that the control plane is highly available.
     When the management cluster worker nodes are spread across different availability zones,
-    the hosted control plane will be spread across different availability zones as well in `PreferredDuringSchedulingIgnoredDuringExecution` mode for
-    `PodAntiAffinity`.
+    the hosted control plane will be spread across different availability zones as well in
+    `PreferredDuringSchedulingIgnoredDuringExecution` mode for `PodAntiAffinity`.
+    If your management cluster doesn't have enough workers (less than 3), you'll need to
```

Comment:
Is this something that is going to be supported? Less than 3 workers on the mgmt cluster?

Reply:
not really but we're dev preview now so better document it for our testers.